### PR TITLE
first version of tests

### PR DESCRIPTION
**Pull request by oleksiistepaniak**

### CODE REVIEW

**Review by OlehSarapuk — APPROVED** [review]

File: src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+import core.basesyntax.exceptions.InvalidDataForRegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static User user;
+    private static StorageDao storageDao;
```
Comment (sarakhmen) [inline src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java:17]: this field isn't used here at all, let's remove it

File: src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    private static User anotherUser;
```
Comment (sarakhmen) [inline src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java:18]: do you really use this anotherUser?

File: src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+        anotherUser = new User();
+        anotherUser.setLogin("bobik");
+        anotherUser.setAge(30);
+        anotherUser.setPassword("idk1488");
+        user = new User();
+        user.setPassword("popbob1337");
+        user.setAge(20);
+        user.setLogin("bobcarlson");
```
Comment (sarakhmen) [inline src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java:31]: when you make your user non-static you can initialize it in @BeforeEach annotated method

**Review by sarakhmen — CHANGES_REQUESTED** [review]
Let's improve your solution :) Also, let's test a successful registration scenario

**Review by nacenik — APPROVED** [review]
Good job) 🔥

**Pull request closed without merging**